src/components/dev-dot-content/mdx-components/mdx-headings/mdx-headings.module.css

```diff
@@ -0,0 +1,61 @@
+.heading {
+  composes: g-offset-scroll-margin-top from global;
+
+  /*
+  Special spacing rule for subsequent list elements.
+  TODO: can we rethink the design spec for spacing to avoid sibling selectors?
+  */
+  & + ul,
+  & + ol {
+    margin-top: 12px;
+  }
+
+  /*
+  Special spacing rule for subsequent paragraph elements.
+  TODO: can we rethink the design spec for spacing to avoid sibling selectors?
+  */
+  & + p {
+    margin-top: 0;
+  }
+
+  /*
+  TODO: these values are a special request from design for the purposes of the
+  inital preview. They make <code> elements within heading elements a little
+  less jarring.
+  */
+  & code {
+    font-size: 0.8em;
+    font-weight: 500;
+  }
+}
+
+/*
+***
+* The following classes are for handling color and margin styles for heading and
+* paragraph elements in DevDot.
+***
+*/
```

Comment:
💭 I think this comment can be removed now since the "paragraph elements" part is no longer accurate and the classnames are now localized to only affect headings.

Reply:
✅  ah great call - removed!